unit_test/test_validate.py

```diff
@@ -3,9 +3,14 @@
 
 
 class TestValidate(TestCase):
-    def test_validate(self):
-        # This assumes a standard environment where everything is
-        # go/src/github.com/rapid7/<projects>
-        directory_to_test = "../../komand-plugins/example/"
+    def test_plugin_validate(self):
```

Comment:
I noticed that the test directories and specs aren't in this repo - those should be included in our repository here.